### ISSUE TITLE
Feature Suggestion: Warn instead of error

### ISSUE THREAD

**Issue by AnthonyWC** [aws-cloudformation/cfn-lint#785]
Currently it is possible to setup the tool so that we can arbitrary suppress checking for certain rules.  However I think there are many situations where we prefer to not skip check entirely but also not to fail the linter if the check does not go through.  We want information so user can make a decision based on output.

Most linters has this type of config option, perhaps maybe something like:
```yaml
Metadata:
  cfn-lint:
    config:
      regions:
      - us-east-1
      - us-east-2
      ignore_checks:
      - E2530
      warn_only:
      - W3005
```

With warn, the error findings should still be emitted but exit code should emit a 0 instead of non-zero if only warn rules were triggered.

**Comment by fatbasstard** [comment]
The exit code system works in this way already, it's not a simple non-zero == error.  This can be seen here: https://github.com/aws-cloudformation/cfn-python-lint/blob/master/src/cfnlint/core.py#L58

**Comment by atkinsonm** [comment]
Sounds like what you would have to do today to have this behave the way you are describing is create a wrapper around the `cfn-lint` call that catches error codes 8 and 4 and transforms them to 0.

That said, I'd probably take advantage of a metadata tag or command line argument like `--warn-only` if support existed.

**Comment by PatMyron** [comment]
discussed here as well: https://github.com/aws-cloudformation/cfn-python-lint/issues/1586#issuecomment-642984414

**Comment by kddejong** [comment]
This is now configurable in v0.90.0 which allows you to configure how the exit code process works.  See PR #2460 that covers the readme change
